subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -15,7 +15,7 @@ config BT_LLL_VENDOR_NORDIC
 	select ENTROPY_NRF5_BIAS_CORRECTION if ENTROPY_NRF5_RNG
 
 	select BT_HAS_HCI_VS
-	select BT_CTLR_CRYPTO_SUPPORT if !SOC_SERIES_NRF54LX
+	select BT_CTLR_CRYPTO_SUPPORT if !SOC_COMPATIBLE_NRF54LX
```

Comment:
@aescolar `SOC_COMPATIBLE_NRF54LX` is not selected for `SOC_SERIES_NRF54LX` causing build failure.